README.md

```diff
@@ -24,131 +16,119 @@ This is the documentation for our **development** version. For the documentation
 - In just **4.5kb** you get a productive environment:
   - [preact]
   - [preact-router]
-  - 1.5kb of conditionally-loaded polyfills for [fetch](https://github.com/developit/unfetch) & [Promise](https://npm.im/promise-polyfill)
+  - 1.5kb of conditionally-loaded polyfills for [fetch] & [Promise]
 
+### Installation
 
-### Commands
+Prerequisites: [Node.js](https://nodejs.org/en/) required version (> 6.x)
 
-`preact create your-app-name`: create a new app
+```sh
+$ npm install -g preact-cli
+```
 
-`preact init`: create a new app interactively
+### Usage
 
-`preact build`: build an app
+```sh
+$ preact create <template-name> <project-name>
+```
 
-`preact watch`: start a dev server
+Example:
+```sh
+$ preact create default my-project
+```
 
+The above command pulls the template from [preactjs-templates/default], prompts for some information, and generates the project at `./my-project/`.
 
-### Quickstart
+### Official Templates
 
-```sh
-# once and you're good:
-npm i -g preact-cli
+The purpose of official preact project templates are to provide opinionated development tooling setups so that users can get started with actual app code as fast as possible. However, these templates are un-opinionated in terms of how you structure your app code and what libraries you use in addition to preact.js.
 
-# create a new project:
-preact create my-great-app (or) preact init
-cd my-great-app
+All official project templates are repos in the [preactjs-templates organization]. When a new template is added to the organization, you will be able to run `preact create <template-name> <project-name>` to use that template.
 
-# start a live-reload/HMR dev server:
-npm start
+Current available templates include:
 
-# go to production:
-npm run build
-```
+- [default] - Default template with all features.
 
-### Using Yarn
+- [material] - material-ui template using preact-material-components
```

Comment:
"material template...." `material-ui` might be confusing with react's material-ui